### PR TITLE
Fix project cards status

### PR DESCRIPTION
**Pull request by iamfranco**
Previously it didn't show correct status text and color

### CODE REVIEW

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-7)

**Review by romanchukenator — APPROVED** [review]
Looks good! 👍

File: cypress/e2e/project-list.cy.ts [hunk 1]
```diff
@@ -21,18 +27,61 @@ describe("Project List", () => {
...
+
+          // check project card contains correct name and language
+          cy.wrap($el).contains(project.name);
+          cy.wrap($el).contains(expectedValue.language);
+
+          // check number of issues is sibling of total issues label
+          cy.wrap($el)
+            .contains("Total issues")
+            .parent()
+            .contains(project.numIssues);
```
Comment (jkettmann) [inline cypress/e2e/project-list.cy.ts:66]: It makes sense to test that the num issues is rendered with the correct label. But I'd either add a test id with `cy-data` or replace this with the following line as that would be more independent of the exact DOM structure

```
cy.wrap($el).contains(`Total issues${mockProjects[index].numIssues}`);
```

File: features/projects/components/project-card/project-card.tsx [hunk 2]
```diff
@@ -16,10 +15,16 @@ const languageNames = {
   [ProjectLanguage.python]: "Python",
 };
 
+const statusNames = {
```
Comment (jkettmann) [inline features/projects/components/project-card/project-card.tsx:18]: Great job 👍

**Pull request merged**